src/tasks/semantic-release/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const fs = require('fs');
+const { MrmError, packageJson, lines, yaml, markdown, uninstall } = require('mrm-core');
+
+const PACKAGE_NAME = 'semantic-release';
+
+function hasSemanticRelease() {
+	const commands = yaml('.travis.yml').get('after_success');
+	return commands && commands.find(cmd => cmd.includes(PACKAGE_NAME)).length > 0;
+}
+
+module.exports = function(config) {
+	// Require .travis.yml
+	if (!fs.existsSync('.travis.yml')) {
+		throw new MrmError(
+			`Run travis task first:
+  mrm travis`
+		);
+	}
+
+	// package.json
+	const pkg = packageJson('package.json');
+
+	if (!hasSemanticRelease()) {
+		throw new MrmError(
+			`Setup semantic-release first:
+  npm i -g semantic-release-cli@latest && semantic-release-cli setup
```

Comment:
`npx semantic-release-cli setup`? No need to keep it installed and you’ll always have the latest version ;-)

Reply:
Thought about that but wasn't sure if we should check the version of npm. But I agree, let's change this.